lib/engine/apps/app_executor.ts

```diff
@@ -207,12 +209,18 @@ export default class AppExecutor extends events.EventEmitter {
         this.mainOutput = new QueueOutputDelegate();
         this._notificationOutput = new NotificationOutputDelegate(this);
         this.notifications = meta.notifications;
+
+        this._date = new Date;
```

Comment:
This needs to be persisted on disk, otherwise it will be incorrect when the engine restarts.